diff --git a/addons/mrp/mrp.py b/addons/mrp/mrp.py
index 012a450..9f4db30 100644
--- a/addons/mrp/mrp.py
+++ b/addons/mrp/mrp.py
@@ -1045,8 +1045,8 @@ class mrp_production(osv.osv):
                                  destination_location_id=source_location_id, context=context)
                     self._make_production_line_procurement(cr, uid, line, shipment_move_id, context=context)
 
-            if shipment_id:
-                wf_service.trg_validate(uid, 'stock.picking', shipment_id, 'button_confirm', cr)
+            # if shipment_id:
+            #     wf_service.trg_validate(uid, 'stock.picking', shipment_id, 'button_confirm', cr)
             production.write({'state':'confirmed'}, context=context)
         return shipment_id
 
diff --git a/addons/stock/stock.py b/addons/stock/stock.py
index 26647d8..2815cf2 100644
--- a/addons/stock/stock.py
+++ b/addons/stock/stock.py
@@ -707,7 +707,7 @@ class stock_picking(osv.osv):
             ("invoiced", "Invoiced"),
             ("2binvoiced", "To Be Invoiced"),
             ("none", "Not Applicable")], "Invoice Control",
-            select=True, required=True, readonly=True, track_visibility='onchange', states={'draft': [('readonly', False)]}),
+            select=True, required=True, readonly=True, track_visibility='onchange'),
         'company_id': fields.many2one('res.company', 'Company', required=True, select=True, states={'done':[('readonly', True)], 'cancel':[('readonly',True)]}),
     }
     _defaults = {
